api/XMLHttpRequestEventTarget.json

```diff
@@ -1,396 +0,0 @@
-{
-  "api": {
-    "XMLHttpRequestEventTarget": {
```

Comment:
I think we should still keep this entry, since there is an interface by this name, and `self.XMLHttpRequestEventTarget` does exist. It won't have any subfeatures though.

Reply:
Makes sense. Added back